src/interfaces/IOracleWriter.sol

```diff
@@ -2,14 +2,19 @@ pragma solidity 0.8.13;
 
 interface IOracleWriter {
     struct Observation {
-        // natural log (ln) of the price (token1/token0)
-        int112 logAccPrice;
-        // natural log (ln) of the liquidity (sqrt(k))
-        int112 logAccLiquidity;
+        // natural log (ln) of the raw accumulated price (token1/token0)
+        int112 logAccRawPrice;
+        // natural log (ln) of the clamped accumulated price (token1/token0)
+        // even in the case of extreme prices (3e75), will overflow once every 878 years
```

Comment:
Is 3e75 the extremity of the price?

Reply:
Can you include 3e75 shown as 2**N so that we can see the relationship between this number and the size of liquidity in bits?

Reply:
alright so the `LogExpMath` lib can take in really large numbers (at the limit of `int256.max` (around `5e77`)) for compressing a raw value into an exponent. But the decompression function has a limit of around `e ** (130) == 2.87e56`. 

When I wrote the comment I wanted to show that even with values beyond the supported range, the accumulator would take too long to overflow. 


So for the sake of proper and sane documentation, we can

Reply:
Yes please

Reply:
okay changed the documentation. The docs describing the liquidity accumulator is not implemented yet, but will be done in #105